lib/matplotlib/figure.py

```diff
@@ -2238,6 +2238,34 @@ def _redo_transform_rel_fig(self, bbox=None):
         self.bbox_relative.p0 = (x0, y0)
         self.bbox_relative.p1 = (x0 + dx, y0 + dy)
 
+    
+    def get_subplotparams(self):
+            return self.subplotpars
+
+    def set_subplot_params(self, subplot_params=None):
```

Comment:
The getter and setter are inconsistent in `_subplotparams` or `_subplot_params`

Additionally, while I think the more spelled out name is _better_, the idea is I think to match the variable as in `__init__`, so they should probably be `[gs]et_subplotpars`, that or have the canonical as `[gs]et_subplotparams` and use `alias` to make `[gs]et_subplotpars` work as well.

I will note that the issue had the more spelled out name, admittedly.